backbone/README.md

```diff
@@ -159,6 +160,35 @@ return MyItemView;
 
 **[⬆ back to top](#table-of-contents)**
 
+## Ordering
+
+  - **Outside** `Marionette.View` definition
+
+  0. `dependencies` requirement
+  0. `static` methods
+  0. `config` objects
+
+  - Ordering for `Marionette.View`:
```

Comment:
what you did is OK. I was suggesting adding between `(and the remaining life cycle methods)` so we don't have to mention every method. only the ones we care about (as you did)